modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/IgniteBinaryMetadataUpdateNodeRestartTest.java

```diff
@@ -113,13 +108,9 @@ public void testNodeRestart() throws Exception {
         for (int i = 0; i < 10; i++) {
             log.info("Iteration: " + i);
 
-            client = false;
-
             startGridsMultiThreaded(SRVS);
 
-            client = true;
-
-            startGrid(SRVS);
+            startClientGrid(SRVS);
```

Comment:
The following `startGrid(SRVS + CLIENTS);` also starts a client nodes.

Reply:
Fixed